lib/charms/ovn_charm.py

```diff
@@ -1432,3 +1434,19 @@ def assess_exporter(self):
         else:
             snap.install('prometheus-ovs-exporter', channel=channel)
         snap.connect_all()
+
+    def restart_exporter(self, only_inactive=False):
+        """Restart OVS prometheus exporter service.
+
+        :param only_inactive: If set to True, and the service is already
+                              active, it wont' be restarted.
+        :type only_inactive: bool
+        :returns: None
+        """
+        restart_action = (ch_core.host.service_start
+                          if only_inactive
+                          else ch_core.host.service_restart)
+
+        ch_core.hookenv.log(
+            "Restarting OVS exporter", level=ch_core.hookenv.INFO)
```

Comment:
In the case of the service already running, this log message would be incorrect?